client-mr/src/main/java/org/apache/hadoop/mapreduce/RssMRConfig.java

```diff
@@ -55,6 +55,12 @@ public class RssMRConfig {
           MR_RSS_CONFIG_PREFIX + RssClientConfig.RSS_DATA_TRANSFER_POOL_SIZE;
   public static final int RSS_DATA_TRANSFER_POOL_SIZE_DEFAULT_VALUE =
           RssClientConfig.RSS_DATA_TRANFER_POOL_SIZE_DEFAULT_VALUE;
+  // Commit sender pool size
```

Comment:
Could you remove this comment? The comment don't give me extra information to help me understand the code.

Reply:
Got it.